shared/utils/build-all-remediation-roles.py

```diff
@@ -233,6 +250,9 @@ def builder():
                 role_src = generate_role_for_input_content(
                     input_path, benchmark_id, profile_id, args.template
                 )
+
+                if args.extension.endswith("yml"):
```

Comment:
I think it should check for `args.template == "urn:xccdf:fix:script:ansible"`  as well.